spec/integration/workbook-xlsx-reader.spec.js

```diff
@@ -76,9 +76,9 @@ describe('WorkbookReader', function() {
   });
   
   describe('edit styles in existing file', function(){
-    beforeEach(async function(){
+    beforeEach(function(){
```

Comment:
why? I quite like moving things to async/await. the test suite runs on node, so there's no transpiling here

Reply:
we currently support node v6 :(
https://travis-ci.org/exceljs/exceljs/builds/479501545?utm_source=github_status&utm_medium=notification

Reply:
damn, right you are :(